CommonTools/RecoUtils/src/precompile.h

```diff
@@ -0,0 +1,13 @@
+// for integration test 
```

Comment:
actually removing just the line for TrackExtra.h makes no difference - are we pulling in all headers from a package in some way?